app/src/main/java/com/onboarding/parkingsystemjava/fragment/SetParkingLotsDialog.java

```diff
@@ -0,0 +1,70 @@
+package com.onboarding.parkingsystemjava.fragment;
+
+import android.app.DialogFragment;
+import android.content.Context;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+
+import androidx.annotation.Nullable;
+
+import com.onboarding.parkingsystemjava.R;
+import com.onboarding.parkingsystemjava.mvp.contract.SetParkingContract;
+import com.onboarding.parkingsystemjava.mvp.presenter.SetParkingPresenter;
+import com.onboarding.parkingsystemjava.mvp.view.SetParkingView;
+
+public class SetParkingLotsDialog extends DialogFragment {
```

Comment:
please do not use set or get on a class name

Reply:
you can change it by a ConfigureParkingSize for example ;)